compiler/rustc_borrowck/src/region_infer/opaque_types.rs

```diff
@@ -378,10 +378,6 @@ fn check_opaque_type_parameter_valid(
     // Only check the parent generics, which will ignore any of the
     // duplicated lifetime args that come from reifying late-bounds.
     for (i, arg) in opaque_type_key.args.iter().take(parent_generics.count()).enumerate() {
-        if let Err(guar) = arg.error_reported() {
-            return Err(guar);
-        }
-
```

Comment:
This is removed to avoid an ICE in `tests/ui/type-alias-impl-trait/under-binder.rs`. I'll rewrite it anyway in #116891.